### PR TITLE
tests: check schema drift from openapi schema

### PR DESCRIPTION
**Pull request by mscolnick**
Tests to check that marimo's backend is up-to-date with the open-api schema (minus the version)

### CODE REVIEW

File: openapi/api.yaml [hunk 1]
```diff
@@ -876,6 +883,7 @@ components:
               type: string
           required:
           - width
+          - auto_download
```
Comment (dmadisetti) [inline openapi/api.yaml:886]: Oh cool, did it catch something?

**Review by dmadisetti — APPROVED** [review]
Nice! Are these going to go into the FE CI?

**Comment by dmadisetti** [comment]
Oh ok. Saw the other PR. Kind of annoying to have to rely on a 3rd part action

File: openapi/api.yaml (hunk 1 above)
Comment (mscolnick) [reply openapi/api.yaml:886]: yep it did! wasnt a bug, but was nice to validate it worked on the first try

**Comment by mscolnick** [comment]
> Are these going to go into the FE CI?

These don't need to go in the frontend CI. If there are changes to this file, then the FE ci will kick off. and this forces the backend to make changes if they exist. So the contract, as expected, is this yaml file.

**Pull request merged**